realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -1973,6 +1973,19 @@ private OsResults lazyFindAll() {
         return this;
     }
 
+    /**
+     * FIXME Add description
+     * FIXME Figure out an API for providing arguments, probably just re-use `String.format()` arguments
+     *
+     * @param filter
+     * @return
+     */
+    public RealmQuery<E> rawPredicate(String filter) {
```

Comment:
I suggested `filtered()` - like Realm JS.

Reply:
The issue I have with `filter` or `filtered` is that it is too generic. I mean it is a great keyword if is the only keyword you have when querying, but in our case, you can interleave with a lot of other statements and there it feels a little weird (at least to me):

```
realm.where(Person.class)
  .equalTo("name", "Jane")
  .filtered("children@count > 2")
  .lessThan("age", 35)
  .findAll();
```


```

Reply:
Not to mention, then `sort()` should be called `sorted()`. And it only makes sense if you don't have *any* of the other keywords (lessThan, equalTo, ...)